Author: Nicolas Dufresne <nicolas.dufresne@gmail.com>
Description: Fix missing return value to non-void return statement.

diff -Naurp squashfs-tools.orig/squashfs-tools/unsquash-4.c squashfs-tools/squashfs-tools/unsquash-4.c
--- squashfs-tools.orig/squashfs-tools/unsquash-4.c	2009-10-02 19:26:35.000000000 +0200
+++ squashfs-tools/squashfs-tools/unsquash-4.c	2009-12-04 08:20:36.000000000 +0100
@@ -38,7 +38,7 @@ int read_fragment_table_4()
 		sBlk.fragment_table_start);
 
 	if(sBlk.fragments == 0)
-		return;
+		return TRUE;
 
 	if((fragment_table = malloc(sBlk.fragments *
 			sizeof(squashfs_fragment_entry))) == NULL)
